packages/server/modules/workspaces/repositories/users.ts

```diff
@@ -80,10 +91,35 @@ export const getInvitableCollaboratorsByProjectIdFactory =
     if (cursor) {
       query.andWhere(Users.col.createdAt, '<', cursor)
     }
-    return await query
+    const res = await query
```

Comment:
this query was busted and wasn't returning the data that the return type said it would